socketio/Cargo.toml

```diff
@@ -7,7 +7,11 @@ description = "An implementation of a socketio client written in rust."
 readme = "../README.md"
 repository = "https://github.com/1c3t3a/rust-socketio"
 keywords = ["socketio", "engineio", "network", "protocol", "client"]
-categories = ["network-programming", "web-programming", "web-programming::websocket"]
+categories = [
+    "network-programming",
+    "web-programming",
+    "web-programming::websocket",
+]
 license = "MIT"
```

Comment:
Is there any good reason you change the formatting in this file? If you foilow a special linter would be nice to know which one and maybe include a config? Or don't reformat?